cmd/wallet-web/src/pages/Signin.vue

```diff
@@ -18,6 +18,12 @@
   >
     <div class="flex flex-col flex-grow justify-center items-center">
       <!--Trustbloc Sign-up provider div -->
+      <toast-notification
+        v-if="systemError"
+        :title="i18n.errorToast.title"
+        :description="i18n.errorToast.description"
```

Comment:
Translations won't get loaded. There is no `errorToast` field in the `src/translations/en|fr.json` for the `Signin` screen.

Reply:
translation files didnt get pushed earlier. Thanks updated